CompositeUi/src/scss/variables/_grid.scss

```diff
@@ -1,42 +0,0 @@
-$small-breakpoint: 640 !default;
```

Comment:
Instead of removing import the variables from node_modules and customize required sizes here, otherwise we are spreading the grid dependency across the whole project